### ISSUE TITLE
`TALK_SMTP_FROM_ADDRESS` configuration

### ISSUE THREAD

**Issue by tomascrespo** [coralproject/talk#2208]
In [documentation ](https://docs.coralproject.net/talk/advanced-configuration/) put this example of SMTP configuration:

`TALK_SMTP_FROM_ADDRESS="The Coral Project" <support@coralproject.net>`

However if I use that format like this:
`TALK_SMTP_FROM_ADDRESS="Verde Pistacho" <comentariosverdepistacho@gmail.com>`

I get this error
```
  err: Error: Invalid sender "<comentariosverdepistacho@gmail.com>"
      at SMTPConnection._formatError (./node_modules/smtp-connection/lib/smtp-connection.js:528:15)
      at SMTPConnection._setEnvelope (./node_modules/smtp-connection/lib/smtp-connection.js:723:30)
      at SMTPConnection.send (./node_modules/smtp-connection/lib/smtp-connection.js:383:10)
      at SMTPTransport.<anonymous> (./node_modules/nodemailer-smtp-transport/lib/smtp-transport.js:126:24)
      at ./node_modules/nodemailer-smtp-transport/lib/smtp-transport.js:162:21
      at SMTPConnection._actionAUTHComplete (./node_modules/smtp-connection/lib/smtp-connection.js:1236:5)
      at SMTPConnection.<anonymous> (./node_modules/smtp-connection/lib/smtp-connection.js:319:22)
      at SMTPConnection._processResponse (./node_modules/smtp-connection/lib/smtp-connection.js:669:16)
      at SMTPConnection._onData (./node_modules/smtp-connection/lib/smtp-connection.js:493:10)
      at emitOne (events.js:116:13)
      at TLSSocket.emit (events.js:211:7)
      at addChunk (_stream_readable.js:263:12)
      at readableAddChunk (_stream_readable.js:250:11)
      at TLSSocket.Readable.push (_stream_readable.js:208:10)
      at TLSWrap.onread (net.js:601:20)
  version: "4.6.7"
  origin: "jobs:mailer"
```

But if I use any of these two formats, sending emails works perfectly
`TALK_SMTP_FROM_ADDRESS=comentariosverdepistacho@gmail.com`
`TALK_SMTP_FROM_ADDRESS="Verde Pistacho" comentariosverdepistacho@gmail.com`

```

**Comment by kgardnr** [comment]
Thank you @tomascrespo! This is great feedback. We'll get that fixed in our docs.

**Comment by immber** [comment]
I've added this change to open Doc's PR https://github.com/coralproject/talk/pull/2249

**Comment by kgardnr** [comment]
Fixed in #2249
